### ISSUE TITLE
default model

### ISSUE THREAD

**Issue by SantaFlamel** [jcjohnson/fast-neural-style#49]
Hi!
 For the creation of default models that are laid out in your article, you use the same default settings for all styles?

**Comment by jcjohnson** [comment]
No, you typically need to play with `-style_image_size`, `-content_weights`, and `-style_weights` to get good results.
